.github/workflows/labeler.yml

```
@@ -0,0 +1,23 @@
+#
+# GitHub action to add labels to pull requests.
+#
+# Label rules are configured in .github/labeler.xml.
+#
+# See https://github.com/marketplace/actions/labeler
+#
+
+name: Pull Request Labeler
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  label:
+    name: Labeler
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v5
```

Labeler v5 has a much more complex config https://github.com/actions/labeler#breaking-changes-in-v5, suggest we stick with v4 to get this running easily.

```suggestion
    - uses: actions/labeler@v4
```